cfg/global_mapper.yaml

```diff
@@ -1,2 +1,40 @@
-test_param: true
+# voxel map
+voxel_map:
+  # min coordinates in world frame [m]
+  x0: 0.0
+  y0: 0.0
+  z0: 0.0
 
+  # max coordinates in world frame
+  x1: 50.0
+  y1: 50.0
+  z1: 10.0
```

Comment:
Yeah.  I agree, but was copying the notation in occ_map.  At this early stage of the project, there is really no excuse for propagating a bad design decision.  Fixed in eb11e12678fa5411ee697310ab18135687010996.